polymetis/tests/scripts/5_continuous_grasping_hw.py

```diff
@@ -54,33 +58,46 @@ def reset_policy(self):
 
         # Send PD controller
         joint_pos_current = self.arm.get_joint_angles()
-        policy = toco.policies.JointImpedanceControl(
+        policy = toco.policies.CartesianImpedanceControl(
             joint_pos_current=joint_pos_current,
-            Kp=self.arm.metadata.default_Kq,
-            Kd=self.arm.metadata.default_Kqd,
+            Kp=KP_DEFAULT,
+            Kd=KD_DEFAULT,
```

Comment:
Do we not want to grab `Kp, Kd` it from the metadata? (i.e. update the config with the values in `KP_DEFAULT`, `KD_DEFAULT`)

Reply:
Adding this to metadata would be a large change that would see very little use as of now.
I implemented it anyways, do you think we should keep it?

Reply:
Might as well do it now rather than later imo